app_dart/README.md

```diff
@@ -14,6 +14,7 @@ and authenticate yourself by running:
 gcloud auth login
 gcloud init
 ```
+
 * [Install Flutter](https://flutter.dev/docs/get-started/install )
```

Comment:
Can we just re-run the build from the dashboard?

Reply:
I'm not entirely sure. Will that use the changes made to the recipe? It is passing here but the dashboard is not updated. Let me try and rerun it.

Reply:
Rerunning. I will abandon this if that succeeds.

Reply:
It has passed the checkout phase. I will close this.